src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -617,3 +644,39 @@ def _parse_capabilities_string(self, capabilities_string):
             desired_capabilities[key.strip()] = value.strip()
 
         return desired_capabilities
+
+
+class CapabilitiesConfig(object):
+
+    def __init__(self, arg):
+        for k in arg:
+            self.__setattr__(k, arg[k])
```

Comment:
It seems you don't understand what `self.__dict__.update(capabilities)` actually does.

Also, I don't think it is a good idea that the keyword returns either an object or a dictionary. If returning a dictionary is needed, it should *always* return a dict that allows accessing keys as attributes.